frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings/SdkInfoAlert.tsx

```diff
@@ -0,0 +1,60 @@
+import { match } from "ts-pattern";
+import { jt, t } from "ttag";
+
+import { useSetting } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { PLUGIN_EMBEDDING_SDK } from "metabase/plugins";
+import { Alert, Icon, Text } from "metabase/ui";
+
+import { useEmbeddingSettingsLinks } from "./links";
+
+export const SdkInfoAlert = () => {
+  const { implementJwtUrl, switchMetabaseBinariesUrl, upgradeUrl } =
+    useEmbeddingSettingsLinks();
+  const isHosted = useSetting("is-hosted?");
+  const isEE = PLUGIN_EMBEDDING_SDK.isEnabled();
+
+  const baseText = t`You can test Embedded analytics SDK on localhost quickly by using API keys. To use the SDK on other sites, `;
+  const upgradeLink = (
+    <ExternalLink
+      key="upgrade"
+      href={upgradeUrl}
+    >{t`upgrade to Metabase Pro`}</ExternalLink>
+  );
+  const jwtLink = (
+    <ExternalLink
+      key="jwt"
+      href={implementJwtUrl}
+    >{t`implement JWT SSO`}</ExternalLink>
+  );
+
+  const alertText = match({ isEE, isHosted })
```

Comment:
this was the massive match statement in the original `EmbeddingSdkSettings` component. I did some boolean table logic and managed to get it down to this.

Reply:
To elaborate on this, in the old code (which you should be able to see [here](https://github.com/metabase/metabase/pull/47929/files#diff-9615a4d60d919495d4a97be577e2444d0c5a3636ce37d6b1137765b514647fbeL95)), we explicitly define the case in which `isEE` is true, but I've made that the default case here (in the `otherwise` block). The other two cases are only based on `isEE=false` and `isHosted` so I figured it would make sense to make the code a little smaller

Reply:
I agree with @WiNloSt, even some comments would be fine with me.
If I understood the code correctly the there cases should be:
- `.with({ isEE: false, isHosted: false },` -> OSS selfhosted
- `.with({ isEE: false, isHosted: true },` ->  starter
- `otherwise()` -> pro or enterprise on either cloud or selfhosted